docs/product-editor-development/product-editor-extensibility-guidelines.md

```diff
@@ -0,0 +1,97 @@
+There are several ways to extend the new product form: from a single field to a whole section containing multiple fields and tables. These extension points are tightly linked to the form structure, giving you plenty of freedom to create a great user experience.
+
+Like everything in the new product form, each extension point is a separate block. However, each existing block can be extended the same as the Block Editor, so for the sake of a consistent and smooth user experience, try to integrate your extension into an existing block before creating a new one.
+
+## Product form tabs
+
+The new product form consists of areas displayed as tabs. Each is a separate view and may contain any number of sections. All areas serve a specific purpose, allowing merchants to quickly find the information they’re looking for (both in default Woo features and extensions).
+
+* **General**: Essential product information, including the name, image, and description. This tab is also where key features live for non-standard product types: downloads, groups, links, etc.
+
+* **Organization**: This tab contains all the data used to organize and categorize product information: from categories to attributes. Best for extensions that provide new ways to describe products, e.g., product identifiers, statuses, special tags, etc.
+
+* **Pricing**: List price, sale price, and tax options. Best for extensions that allow merchants to set up additional payment methods (e.g., Subscriptions) or add advanced pricing schemes, like wholesale.
+
+* **Inventory**: Basic inventory settings and stock management options. Here merchants come to update the quantity at hand or mark the product as out of stock. Best for extensions that enable conditional inventory management, dropshipping options, or various restrictions.
+
+* **Shipping**: All the information merchants need to enter to present customers with accurate shipping rates at checkout. Best for physical product details that may impact shipping (e.g. capacity or volume), additional shipping carrier settings, or custom shipping options.
+
+* **Variations**: Contains product options and variations. It is the only tab that doesn’t support extensibility in its current form (you can add your extension to the variation details page as well as provide custom update actions triggered from the table view or the bulk editor).
+
+## Where should your extension go?
+
+Depending on the type of your extension (and your use case), you can pick an interface location that best suits how users will interact with it.
+
+To choose, put yourself in the merchant’s shoes: where would you go to find this feature? What is it related to? Adding your extension to a group of similar features will help merchants navigate the form and make it easier for them to find your extension.
+
+### Field 
+
+Field extensions are the simplest type of extension. They can add new, replace, and expand existing form fields in any section, hide fields, and change form defaults.
```

Comment:
"change form defaults" -- do we support this? How?

Reply:
I will remove that part. Currently, to do that I think the user would have to remove the default block and add a new one with the desired behavior.